data/packages.xml

```diff
@@ -486,8 +485,11 @@
       <pkgname>splix</pkgname>
     </feature>
     <feature name="firefox">
-        <pkgname conflicts="chromium,chromium-pepper-flash">firefox</pkgname>
-        <pkgname conflicts="chromium,chromium-pepper-flash">flashplugin</pkgname>
+        <pkgname conflicts="chromium">firefox</pkgname>
+    </feature>
+    <feature name="flash">
+        <pkgname conflicts="chromium-pepper-flash">flashplugin</pkgname>
+        <pkgname conflicts="flashplugin">chromium-pepper-flash</pkgname>
```

Comment:
@Wyn10 : Nope, that won't work. How does cnchi know if it has to install chromium-pepper-flash or flashplugin? Is there any problem if we install both packages?